README.md

```diff
@@ -107,4 +107,4 @@ UNLEASH_API_URL=http://127.0.0.1:4242/api \
 
 or similar. The functional test suite looks for a manually setup set of
 features. E.g. log into the Unleash UI on port 4242 and create a feature called
-`default`.
+`default` & `semver` with a `SEMVER_EQ` strategy constraint.
```

Comment:
Imo it would make more sense to mock this using https://github.com/alexliesenfeld/httpmock for example, than having to do manual steps to run tests.

Reply:
Thats the *functional* test suite, which exists deliberately to check consistency with the actual separate publish Unleash API server code, which varies and changes independently. There are unit tests separate to that.